ext/transaction-processes/default-booking/templates/booking-accepted-request/booking-accepted-request-html.html

```diff
@@ -22,7 +22,7 @@
               <td>
                 <h1 style="color:#484848;font-size:26px;line-height:1.3;font-weight:700">{{t "BookingAcceptedRequest.Title" "Your booking request was accepted"}}</h1>
                 {{#each tx-line-items}}{{#eq "line-item/hour" code}}
-                <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848"><p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">{{t "BookingAcceptedRequest.DescriptionHourly" "{providerDisplayName} has accepted your booking request for {listingTitle} from {dateStart,date,::hmmaYYYYMMMd} to {dateEnd,date,::hmmaYYYYMMMd}." dateStart=booking.start dateEnd=booking.end listingTitle=listing.title providerDisplayName=provider.display-name}}</p></p>
+                <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848"><p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">{{t "BookingAcceptedRequest.DescriptionHourly" "{providerDisplayName} has accepted your booking request for {listingTitle} from {dateStart,date,::jmmYYYYMMMd} to {dateEnd,date,::jmmYYYYMMMd}." dateStart=booking.start dateEnd=booking.end listingTitle=listing.title providerDisplayName=provider.display-name}}</p></p>
```

Comment:
This wasn't introduced by this PR but caught my eye: there's a nested <p> tag here which looks like an accidental duplicate

`  <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848"><p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">`

Reply:
Nicely spotted, I fixed it now in core as well.